src/rtp_transceiver/rtp_receiver/mod.rs

```diff
@@ -27,19 +27,29 @@ use tokio::sync::{watch, Mutex, RwLock};
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 #[repr(u8)]
 pub enum State {
-    Initial = 0,
-    Running = 1,
-    Paused = 2,
-    Closed = 3,
+    /// We haven't started yet.
```

Comment:
```suggestion
    /// The transceiver hasn't started yet.
```

Reply:
This state isn't transceiver specific, it's receiver specific. Further, I could see us using this struct for RtpSender too down the line

Reply:
I'm thinking of "we" as which ever object is using this state enum to track its current state. Happy to hear other suggestions or perhaps I can reword it to no longer include the word "we".